GenericDockerfile

```diff
@@ -29,7 +29,7 @@ LABEL org.label-schema.description="System configuration data for the CARMA Plat
 LABEL org.label-schema.vendor="Leidos"
 LABEL org.label-schema.version=${VERSION}
 LABEL org.label-schema.url="https://highways.dot.gov/research/research-programs/operations/CARMA"
-LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/CARMAPlatform"
+LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-config"
```

Comment:
Are we sure this is not supposed to be carma-platform?

Reply:
i think the version control system url is supposed to point to that component's repo. At least that's how the rest of them are structured.